sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -507,6 +507,166 @@ public partial interface IAzureClientFactoryBuilderWithCredential
         Azure.Core.Extensions.IAzureClientBuilder<TClient, TOptions> RegisterClientFactory<TClient, TOptions>(System.Func<TOptions, Azure.Core.TokenCredential, TClient> clientFactory, bool requiresCredential = true) where TOptions : class;
     }
 }
+namespace Azure.Core.GeoJson
```

Comment:
I thought the convention was to use only "Azure" for the namespace (or, in this case, "Azure.GeoJson") for types meant to be used by the general dev public, but "Azure.Core(.Whatever)" for types mainly used across our assemblies that can't be shared source.

Reply:
I understand this is to maintain the existing namespace, but since the type name is technically changing anyway (diff assembly) is it worth "fixing" that now?

Reply:
Hm, good point. We have `Azure.Messaging` already. Let me change that and see if anyone objects.